src/core/namespace.c

```diff
@@ -319,8 +328,12 @@ static int apply_mount(
         r = mount(what, m->path, NULL, MS_BIND|MS_REC, NULL);
         if (r >= 0)
                 log_debug("Successfully mounted %s to %s", what, m->path);
-        else if (m->ignore && errno == ENOENT)
-                return 0;
+        else {
+                if (m->ignore && errno == ENOENT)
+                        return 0;
+                else
+                        log_error("Failed mounting %s to %s: %s", what, m->path, strerror(errno));
+        }
```

Comment:
should be `log_error_errno(errno, …` I figure... also the "else" can be dropped, after all the if branch already returned here...